template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/appsettings.json

```diff
@@ -1,67 +1,83 @@
+//        "Kestrel": {
+//          "Endpoints": {
+//            "Localhost": {
+//              "Address": "127.0.0.1",
+//              "Port": "55555"
+//            },
+//            "LocalhostHttps": {
+//              "Address": "127.0.0.1",
+//              "Port": "43434",
+//              "Certificate": "HTTPS"
+//            }
+//          }
+//        }
 //      },
-//      "LocalhostHttps": {
-//        "Address": "127.0.0.1",
-//        "Port": "43434",
-//        "Certificate": "HTTPS"
-//      }
-//    }
-//  },
 //#if (IndividualLocalAuth)
-//  "IdentityService": {
-//    "Issuer": "https:////localhost/09732173-2cef-46b7-83db-1334bcb079d3/v2.0/",
-//    "SigningCertificates": "IdentityService"
-//  },
+//      "IdentityService": {
+//        "Issuer": "https:////localhost/09732173-2cef-46b7-83db-1334bcb079d3/v2.0/",
+//        "SigningCertificates": "IdentityService"
+//      },
 //#endif
-//  "Authentication": {
+//      "Authentication": {
 //#if (IndividualLocalAuth)
-//    "IdentityService": {
-//      "ClientId": "53bc9b9d-9d6a-45d4-8429-2a2761773502",
-//      "TokenRedirectUrn": "urn:self:aspnet:identity:integrated"
-//    }
+//        "Schemes": {
+//          "IdentityService": {
+//            "ClientId": "53bc9b9d-9d6a-45d4-8429-2a2761773502",
+//            "TokenRedirectUrn": "urn:self:aspnet:identity:integrated"
+//          }
+//        }
 //#elseif (SingleOrgAuth)
-//    "AzureAd": {
-//      "AzureAdInstance": "https:////login.microsoftonline.com/",
-//      "ClientId": "11111111-1111-1111-11111111111111111",
+//        "DefaultSignInScheme": "Cookies",
+//        "DefaultAuthenticateScheme": "Cookies",
+//        "DefaultChallengeScheme": "OpenIdConnect",
+//        "AzureAd": {
+//          "AzureAdInstance": "https:////login.microsoftonline.com/",
+//          "ClientId": "11111111-1111-1111-11111111111111111",
 //#elseif (MultiOrgAuth)
-//    "AzureAd": {
-//      "AzureAdInstance": "https:////login.microsoftonline.com/common",
-//      "ClientId": "11111111-1111-1111-11111111111111111",
+//        "DefaultSignInScheme": "Cookies",
+//        "DefaultAuthenticateScheme": "Cookies",
+//        "DefaultChallengeScheme": "OpenIdConnect",
+//        "AzureAd": {
+//          "AzureAdInstance": "https:////login.microsoftonline.com/common",
+//          "ClientId": "11111111-1111-1111-11111111111111111",
 //#elseif (IndividualB2CAuth)
-//    "AzureAdB2C": {
-//      "AzureAdB2CInstance": "https:////login.microsoftonline.com/tfp",
-//      "ClientId": "11111111-1111-1111-11111111111111111",
-//      "CallbackPath": "/signin-oidc",
-//      "Domain": "qualified.domain.name",
-//      "SignUpSignInPolicyId": "MySignUpSignInPolicyId",
-//      "ResetPasswordPolicyId": "MyResetPasswordPolicyId",
-//      "EditProfilePolicyId": "MyEditProfilePolicyId"
-//    }
+//        "DefaultSignInScheme": "Cookies",
+//        "DefaultAuthenticateScheme": "Cookies",
+//        "DefaultChallengeScheme": "OpenIdConnect",
```

Comment:
I don't think the OpenIdConnect scheme is registered anymore given these changes. Should this be AzureAdB2CDefaults.SignUpSignInAuthenticationScheme?